File: src/main/java/com/herobrine/future/enchantment/EnchantmentChanneling.java
package com.herobrine.future.enchantment;

public class EnchantmentChanneling extends EnchantmentTridentBase {
    public EnchantmentChanneling() {
        super(Rarity.VERY_RARE, "channeling");
    }

    @Override
    public int getMinEnchantability(int enchantmentLevel) {
        return 25;
    }

    @Override
    public int getMaxEnchantability(int enchantmentLevel) {
        return 50;
    }

    @Override
    public int getMaxLevel() {
        return 1;
    }
}

File: src/main/java/thedarkcolour/futuremc/enchantment/EnchantmentChanneling.kt
package thedarkcolour.futuremc.enchantment

class EnchantmentChanneling : EnchantmentTridentBase(Rarity.VERY_RARE, "channeling") {
    override fun getMinEnchantability(enchantmentLevel: Int): Int {
        return 25
    }

    override fun getMaxEnchantability(enchantmentLevel: Int): Int {
        return 50
    }

    override fun getMaxLevel(): Int {
        return 1
    }
}